src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
>If you are increasing the timeout value on HttpClient, then I assume it is because you feel that the test server (3rd party server) is really taking even longer than the 40 seconds we asked it to delay.

My concern isn't for the server taking longer, it's for the tests themselves taking longer when being run concurrently with other test libraries as a part of build.cmd. 

>It sounds like you still want to keep the test semantically the same....i.e. test that having an HttpClient.Timeout value > (the actual delay time of the server) means that the request will succeed. 

Yes, I do.

>The point of this test is to verify that the operation of HttpClient Timeout feature is working as expected. And we choose 60 seconds because it is the non-default value of that property.

The point of this test is specifically to verify that the Timeout does not occur before it is supposed to, yes? We don't ever want to hit the timeout, or you're right that something is wrong and the test should fail. My reasoning is since that is the rare case and *shouldn't* occur in a reliable test, the timeout can be unreasonably long since we will mostly exit long before the timeout is reached.

>And, having a long timeout like that means we will never want this test to be InnerLoop because it will make inner loop tests slow.

Only when it fails. But, yeah, that's an impediment to moving it to Innerloop, though I'm not certain it's a complete blocker.